README.developers.md

```diff
@@ -661,6 +667,14 @@ The following table provides details on available Koios settings in VTR flow:
 
 For more information refer to the [Koios benchmark home page](vtr_flow/benchmarks/verilog/koios/README.md).
 
+To make running all the koios benchmark easier, especially with thos circuits scattered between different tasks, we added a task list that triggers all the 40 circuits of Koios as follows (this will run all the circuits with cimplex DSP functionality enabled. If you want to disable the complex DSP, edit the file to point to `koios_*_no_hb` tasks):
```

Comment:
benchmark -> benchmarks
cimplex -> complex
to koios_*_no_hb --> to the koios_*_no_hb

Reply:
we added a task list --> there is an overall task list that runs all the 40 circuits of ...